src/styles/base/_accessibility.scss

```diff
@@ -0,0 +1,6 @@
+@media screen and (prefers-reduced-motion: reduce) {
```

Comment:
Interesting!

Reply:
Yeah, have a look at: https://css-tricks.com/introduction-reduced-motion-media-query/ :)

Reply:
It's a great read! Thanks @Jonic!